integration_tests/mockApis/alertsApi.ts

```diff
@@ -0,0 +1,258 @@
+import { stubFor } from './wiremock'
+import {
+  emptyAlertsMock,
+  pagedActiveAlertsMock,
+  pagedActiveAlertsMockFiltered,
+  pagedActiveAlertsMockPage2,
+  pagedActiveAlertsMockSorted,
+  pagedInactiveAlertsMock,
+} from '../../server/data/localMockData/pagedAlertsMock'
+import { alertTypesMock } from '../../server/data/localMockData/alertTypesMock'
+import { alertDetailsExpiresMock, alertDetailsMock } from '../../server/data/localMockData/alertDetailsMock'
+
+export default {
+  stubActiveAlerts: (prisonerNumber: string) => {
+    let jsonResp
+    if (prisonerNumber === 'G6123VU') {
+      jsonResp = pagedActiveAlertsMock
+    } else if (prisonerNumber === 'A1234BC') {
+      jsonResp = emptyAlertsMock
+    }
```

Comment:
Would suggest passing in the payloads to the mock rather than conditional based on prisonerNumber as it makes it easier to read what is happening from within the test